static/app/views/settings/project/projectOwnership/index.tsx

```diff
@@ -238,6 +238,7 @@ tags.sku_class:enterprise #enterprise`;
     const {ownership, codeowners} = this.state;
 
     const disabled = !organization.access.includes('project:write');
+    const editOwnershipeRulesDisabled = !organization.access.includes('project:read');
```

Comment:
New variable to disable/enable just the "Ownership Rules" block. It now just requires `project:read` scope to be able to edit this